py_twitter/PyTwitter.py

```diff
@@ -278,3 +280,63 @@ def geo(self, lat: float, long: float,
         response = self.api.get('geo/search.json', params)
         tweets = response['result']['places']
         return tweets
+
+    def write_tweets(self, tweets: list, file_name: str, file_format: str):
+        """
+        Method that writes a list of twitter ids or tweets dict into json or csv file.
+
+        Params:
+        ----------
+        `tweets (List[str]) or (List[dict]):` TweetsId or TweetsDictionary.
+
+        `file_name (str):` the name of the outputed file.
+
+        `file_format (str):` should be csv or json
+        
+        """
+        if type(tweets[0]) == str:
+            tweets = self.show_lookup(tweets)
+        if file_format == 'json':
+            file = open(file_name + '.json', 'w')
+            json.dump(tweets, file, indent=4)
+            file.close()
+        elif file_format == 'csv':
+            keys = ['created_at', 'id_str', 'full_text', ['entities', 'hashtags'],
+                        'lang', ['user', 'id_str', 'screen_name', 'location'],
+                        'retweet_count', 'favorite_count', 'coordinates']
+            converted_tweets = []
+            for tweet in tweets:
+                converted_tweets.append(self.__selected_keys(keys, tweet))
+
+            with open(file_name + '.csv', 'w') as f:
```

Comment:
There are some errors when running on windows, adding this code solves the problem: `encoding="utf-8", newline="\n"`

Reply:
Sorry for the delay. I just done the commit to the branch and remember stay cheeky breeki my friend.